common/flatpak-dir.c

```diff
@@ -8469,7 +8469,10 @@ flatpak_dir_find_installed_ref (FlatpakDir   *self,
     }
 
   g_set_error (error, FLATPAK_ERROR, FLATPAK_ERROR_NOT_INSTALLED,
-               _("%s %s not installed"), opt_name ? opt_name : "*unspecified*", opt_branch ? opt_branch : "master");
+               _("%s/%s/%s not installed"),
+               opt_name ? opt_name : "*unspecified*",
+               opt_arch ? opt_arch : flatpak_get_arch (),
+               opt_branch ? opt_branch : "master");
```

Comment:
And similarly the `find_matching_ref` call above searches for any branch if none was specified. Maybe just omit the arch section if `opt_arch` is NULL, and omit the branch section if `opt_branch` is NULL?

Reply:
I’ve pushed a fixup which changes them all to `*unspecified*` just like the `opt_name` handling originally did.